pysparkle/pysparkle/data_quality/main.py

```diff
@@ -0,0 +1,47 @@
+import logging
```

Comment:
The changes here are mostly limited to using Pydantic model, but it shows as more differences because of renaming the file.